shiny/workflowPlot/helper.R

```diff
@@ -0,0 +1,69 @@
+library(shiny)
+library(PEcAn.DB)
+library(RPostgreSQL)
+library(dplyr)
+library(lubridate)
+
+bety <-  src_postgres(dbname = "bety", host = "localhost",
```

Comment:
why not read the database settings from config.php rather than hard coding? See for example https://github.com/PecanProject/pecan/blob/master/web/PEcAn.Network.R#L33

This code as is won't run on the BU servers because host isn't localhost